### PR TITLE
Datetime refactor

### PR DESCRIPTION
**Pull request by t-young31**
See https://github.com/inform-health-informatics/Inform-DB/pull/113

### CODE REVIEW

**Review by skeating — COMMENTED** [review]
Did you use emap_doc_generator ?
This generates these tables from the doc strings in InformDB

File: data_dictionary/tables.md [hunk 1]
```diff
@@ -159,20 +150,20 @@ This a single visit to the hospital.
...
-| dischargeTime | timestamp without timezone | Date and time at which this **HospitalVisit** formally ended. |
-| patientClass | varchar(255) | The patient class. E.g. Inpatient or Outpaitent. |
-| arrivalMethod | varchar(255) | The patient's arrival method at hospital. |
+| Name                 | Type | Description |
+|----------------------| --- |---|
+| hospitalVisitId      | bigint | Unique identifier in EMAP for this **HospitalVisit** record. |
+| mrnId                | [Mrn](#Mrn) | Identifier for the [Mrn](#Mrn) associated with this record. |
+| sourceSystem         | varchar(255) | The source system from which we learnt about this hospital visit. |
+| presentationDatetime | timestamp without timezone | Date and time at which this **HospitalVisit** was first recorded. |
+| admissionDaetime     | timestamp without timezone | Date and time at which this **HospitalVisit** formally began. |
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:159]:
```suggestion
| admissionDatetime     | timestamp without timezone | Date and time at which this **HospitalVisit** formally began. |

```

File: data_dictionary/tables.md [hunk 2]
```diff
@@ -159,20 +150,20 @@ This a single visit to the hospital.
...
+| dischargeDaetime     | timestamp without timezone | Date and time at which this **HospitalVisit** formally ended. |
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:160]:
```suggestion
| dischargeDatetime     | timestamp without timezone | Date and time at which this **HospitalVisit** formally ended. |

```

**Review by AnikaC-git — APPROVED** [review]
Thanks ever so much for adding this in. 

Did you check that we don't use any of the changed columns in the SQL queries in the DB cookbook? That is in the internal_docs_emap for now...

File: data_dictionary/tables.md (hunk 1 above)
Comment (stefpiatek) [reply data_dictionary/tables.md:159]: wait, is this not all automatically generated? 

Also, `timestamp without timezone?` All of these should have timezones

**Review by stefpiatek — APPROVED** [review]

**Comment by t-young31** [comment]
> Did you use emap_doc_generator ?

Nope 😓 – didn't know that existed. What's the strategy there?

**Comment by t-young31** [comment]
Will be made with https://github.com/inform-health-informatics/emap_doc_generator on release

**Pull request closed without merging**